pages/index.tsx

```diff
@@ -34,7 +34,7 @@ const Home: NextPage = () => {
             <Section color='light-gray'>
                 <Services />
             </Section>
-            <Section color='primary' paddingBottom=''>
+            <Section color='primary' paddingBottom='' paddingTop=''>
```

Comment:
Is this equivalent to saying `paddingBottom=0`? Might be more clear than passing an empty string?